fairlearn/datasets/_fetch_bank_marketing.py

```diff
@@ -66,7 +66,7 @@ def fetch_bank_marketing(*, cache=True, data_home=None,
         feature_names : list of length 14
             Array of ordered feature names used in the dataset.
         DESCR : string
-            Description of the UCI Adult dataset.
+            Description of the UCI bank marketing dataset.
```

Comment:
I think there are more places in the comment section where this correction can be made.

Example:
1. The number of features mentioned in the description of the dataset is 17 and the comments suggest that it is 14 [here](https://github.com/fairlearn/fairlearn/blob/69eb4b12cbc9c83ee2513186b6d175d5fe81d194/fairlearn/datasets/_fetch_bank_marketing.py#L59) & [here](https://github.com/fairlearn/fairlearn/blob/69eb4b12cbc9c83ee2513186b6d175d5fe81d194/fairlearn/datasets/_fetch_bank_marketing.py#L66).
2. The target/goal of the dataset is not to predict if someone is earning more than $50k or not. This can be corrected [here](https://github.com/fairlearn/fairlearn/blob/69eb4b12cbc9c83ee2513186b6d175d5fe81d194/fairlearn/datasets/_fetch_bank_marketing.py#L63). 
3. Number of datapoints also needs correction at some places (like: [here](https://github.com/fairlearn/fairlearn/blob/69eb4b12cbc9c83ee2513186b6d175d5fe81d194/fairlearn/datasets/_fetch_bank_marketing.py#L59) & [here](https://github.com/fairlearn/fairlearn/blob/69eb4b12cbc9c83ee2513186b6d175d5fe81d194/fairlearn/datasets/_fetch_bank_marketing.py#L62)).

Reply:
@parul100495, apologies for the oversight. I fixed the above issues.

Reply:
No worries. This looks good now 👍